source/opt/instruction.cpp

```diff
@@ -182,8 +182,21 @@ Instruction* Instruction::GetBaseAddress() const {
     }
   }
 
-  assert(base_inst->IsValidBasePointer() &&
-         "We cannot have a base pointer come from this load");
+  switch (opcode()) {
```

Comment:
Need to update the method comment for GetBaseAddress since it currently says "The current instruction must be a load operation".  That has to be generalized.

Reply:
done.